src/main/java/Todo.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Supports the creation of Todo objects.
+ */
+public class Todo extends Task {
+
+    /**
+     * Creates a Todo object
+     * @param description Description of task.
+     * @param taskType Type of task.
+     */
```

Comment:
perhaps it would be good to have an empty line between the descriptions and parameters.